fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/api/client/render/fluid/v1/FluidRenderHandlerRegistry.java

```diff
@@ -39,7 +41,7 @@
 	 * @param fluid The Fluid.
 	 * @return The FluidRenderHandler.
 	 */
-	FluidRenderHandler get(Fluid fluid);
```

Comment:
This could be null, it causes null pointer exceptions when the fluid is the air! So I've marked as @Nullable